onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -595,12 +625,16 @@ class PlannerImpl {
         // OrtValue index of the considered output NodeArg.
         const auto current = Index(node_output->Name());
         AllocPlan(current).value_type = utils::GetMLDataType(*node_output);
+        AllocPlan(current).life_interval.first = program_counter;
+        AllocPlan(current).allocate_interval.first = program_counter;
         // Declare OrtValue index of the reused buffer.
         // The the OrtValue indexed by current may reuse the memory in the OrtValue indexed by reused.
         OrtValueIndex reused;
         if (std::find(graph_outputs.begin(), graph_outputs.end(), node_output) != graph_outputs.end()) {
           // node_output is graph's output, so we can't reuse intermediate buffer
           AllocPlan(current).alloc_kind = AllocKind::kAllocateOutput;
+          AllocPlan(current).life_interval.second = execution_plan.size();
+          AllocPlan(current).allocate_interval.second = execution_plan.size();
```

Comment:
There is no need to add this. I have already added this information and checked-in.

Reply:
As we discussed, I removed the alloc_interval. But kept lifetime_interval. Please help review. Thanks